Pi_Hole_Ad_Blocker/stats.py

```diff
@@ -96,18 +96,24 @@
     except:
       time.sleep(1)
       continue
+        
+#DirtyDaveMcGee edit to improve readability
+#192.168.x.x addresses were being cut off on my display
+#and i wanted to keep the cpu, disk, and memory stats
 
```

Comment:
Don't bother to annotate your changes. If they work for all cases, just make the changes.